javascript/javascript2/week1/spiritanimal.js

```diff
@@ -1,28 +1,31 @@
-const spiritAnimas = [ "fullmoonwolf","The Butterfly","Teen wolf","Spidersense"]
+const spiritAnimals = [
+  "fullmoonwolf",
+  "The Butterfly",
+  "Teen wolf",
+  "Spidersense",
+];
 const button1 = document.getElementById("btn1");
 
- const randomNumber1 = Math.floor(Math.random()*10); 
-    
- const h2 = document.getElementById("spiritinfo");
+const randomNumber1 = Math.floor(Math.random() * 4);
```

Comment:
Instead of 4, please use the length of your array: spiritAnimals.length, then if you add new elements to array, you don't have to change all places when you inserted hardcoded value 4.